index.js

```diff
@@ -10,7 +10,11 @@ const PACKAGE_NAME = require(path.resolve(__dirname, 'package.json')).name;
 const defaultErrorFactory = ({ allowedStatusCodes }) =>
     function errorHandler(error, request, reply) {
         request.log.error(error);
-        if (reply.statusCode === 500) {
+        // @fastify/sensible explicit internal errors support
+        if (
+            reply.statusCode === 500 &&
```

Comment:
Why do you do this check?

Reply:
`@fastify/sensible` has an explicit internal error that we can throw with a custom message (https://github.com/fastify/fastify-sensible/blob/43e01bf259161c64b283df9a5e0ffad33ff8cf8b/lib/httpErrors.js#L139)  . This error adds the special flag `explicitInternalServerError`, which we are checking to avoid that the error message is overridden (only in this case, which is explicit). Basically , it's the same check done [here](https://github.com/fastify/fastify-sensible/blob/48ef8a00ef7b9396a538711b58e7d09b410e1025/index.js#L47).